app/scripts/lib/multichain-api/wallet-getPermissions.js

```diff
@@ -1,20 +1,21 @@
 import { MethodNames } from '@metamask/permission-controller';
```

Comment:
Should this file be in the `multichain-api/` folder?

Reply:
`MethodNames`? I don't believe so. Or at least I'm not understand why it this constant should be copied out of the permission controller package.

If you're talking about the `wallet-getPermissions.js` file itself, it is already in the `multichain-api/` folder